README.md

```diff
@@ -139,30 +163,19 @@ The storage account SKU specifies what type of replication applies to the storag
 
 ```csharp
 // Update storage account sku
-var parameters = new StorageAccountUpdateParameters
-{
-    Sku = new Sku(skuName)
-};
-var storageAccount = storageMgmtClient.StorageAccounts.Update(rgname, acctName, parameters);
+StorageSku updateSku = new StorageSku(StorageSkuName.StandardLrs);
+StorageAccountCreateOrUpdateContent updateParams = new StorageAccountCreateOrUpdateContent(updateSku, kind, location);
+await accountCollection.CreateOrUpdateAsync(WaitUntil.Completed, storAccountName, updateParams);
 ```
 
 Note that modifying the SKU for a production storage account may have associated costs. For example, if you convert a locally redundant storage account to a geo-redundant storage account, you will be charged for replicating your data to the secondary region. Before you modify the SKU for a production account, be sure to consider any cost implications. See [Azure Storage replication](https://azure.microsoft.com/documentation/articles/storage-redundancy/) for additional information about storage replication.
 
-### Check storage account name availability

```

Comment:
Why no Track2 sample for this?
We might can add a tracks SDK sample for this, the API should looks like :
```
subscription.CheckStorageAccountNameAvailability(new StorageAccountNameAvailabilityContent("accountname"));
```